repl/repl.go

```diff
@@ -153,6 +153,8 @@ func (r *REPL) executeLine(t string) (values.Value, error) {
 		return nil, err
 	}
 
+	r.scope.SetReturn(nil)
```

Comment:
Correct me if I'm wrong, but is this necessary? Why not just use the value returned by `interpreter.Eval` and not mess with the scope? `interpreter.Eval` will return any produced side effects. This means TableObjects as well as any value resulting from any expression statements.

Reply:
I had a similar thought, but their could be many returned side effects and there is no consistent way to know which side effect is the result of the expression. So for now I just made the change I knew would fix the issue. 

I'll create a new issue to address using side effects instead of the scope return value for the REPL.

Reply:
Ok sounds good to me.

Reply:
https://github.com/influxdata/flux/issues/867